components/backup-stream/src/subscription_manager.rs

```diff
@@ -598,6 +598,15 @@ where
     }
 
     async fn get_last_checkpoint_of(&self, task: &str, region: &Region) -> Result<TimeStamp> {
+        #[cfg(feature = "failpoints")]
+        fail::fail_point!("get_last_checkpoint_of", |hint| Err(Error::Other(
```

Comment:
Failpoint is only enabled when `failpoint` feature is on. So no need to add `failpoints`. 
```suggestion
        fail::fail_point!("get_last_checkpoint_of", |hint| Err(Error::Other(
```

Reply:
`fail` in the crate is an optional dependency which would only be required when `failpoints` feature enabled. I am afraid if we remove the config it cannot compile...

Reply:
See:
https://github.com/tikv/tikv/blob/21e3bd64e8de517e09836cda4991b446914f3f75/components/backup-stream/Cargo.toml#L11